src/main/java/core/basesyntax/Truck.java

```diff
@@ -0,0 +1,13 @@
+package core.basesyntax;
+
+public class Truck extends Machine {
+    @Override
+    public void doWork() {
+        System.out.println("The truck started working");

```

Comment:
The message 'The truck started working' could be more informative by specifying that it's a Truck. For example, 'Truck is working' would be clearer.